### PR TITLE
feat: Added card view for Room Type

### LINKED ISSUE #9: Booking: App (Room Types)
Should get a list of rooms, that has been requested from the api.

Design: https://www.figma.com/file/sl27ZKzB6GB7hkxxlvjXaY/Overnites?node-id=88%3A316&t=MMVgXasYaHXEtJ0M-0
Depends on: HF5-PROJECT/api#27 HF5-PROJECT/app#8

### PR DESCRIPTION
**Pull request by Anders164a**
## Description

- Added card view for room type

## Type of Change

- [x] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: lib/room/type/view/hotel_room_types.dart [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+              context.l10n.noResultsTryRefreshing,
+            ),
+          );
+        }
+
+        return Scaffold(
+          body: SafeArea(
+            child: Column(
+              children: [
+                HeaderWidget(
```
Comment (frederikpyt) [inline lib/room/type/view/hotel_room_types.dart:83]: image

Comment (Anders164a) [reply lib/room/type/view/hotel_room_types.dart:83]: Changes added

File: lib/app/widgets/header_back.dart [hunk 2]
```diff
@@ -0,0 +1,71 @@
+import 'package:flutter/material.dart';
```
Comment (frederikpyt) [inline lib/app/widgets/header_back.dart]: Wrong way of doing this

Comment (frederikpyt) [reply lib/app/widgets/header_back.dart]: Add a calculation in the real header

**Review by frederikpyt — CHANGES_REQUESTED** [review]

File: lib/app/widgets/header_back.dart (hunk 2 above)
Comment (Anders164a) [reply lib/app/widgets/header_back.dart]: Changes added

**Review by frederikpyt — APPROVED** [review]

**Pull request merged**